### ISSUE TITLE
Can't use it with Browserify and an isomorphic React app

### ISSUE THREAD

**Issue by DWboutin** [RakanNimer/react-google-charts#62]
It give me this error

```/Users/mboutin2/working-directory/project/node_modules/scriptjs/dist/script.js:12
var doc = document
  ^

  ReferenceError: document is not defined
at /Users/mboutin2/working-directory/project/node_modules/scriptjs/dist/script.js:12:13
at doc (/Users/mboutin2/working-directory/project/node_modules/scriptjs/dist/script.js:8:72)
at Object.<anonymous> (/Users/mboutin2/working-directory/project/node_modules/scriptjs/dist/script.js:11:3)
at Module._compile (module.js:413:34)
at Module._extensions..js (module.js:422:10)
at Object.require.extensions.(anonymous function) [as .js] (/Users/mboutin2/working-directory/project/node_modules/babel-register/lib/node.js:166:7)
at Module.load (module.js:357:32)
at Function.Module._load (module.js:314:12)
at Module.require (module.js:367:17)
at require (internal/module.js:20:19)
at Object.<anonymous> (/Users/mboutin2/working-directory/project/node_modules/react-google-charts/lib/components/GoogleChartLoader.js:16:17)
at Module._compile (module.js:413:34)
at Module._extensions..js (module.js:422:10)
at Object.require.extensions.(anonymous function) [as .js] (/Users/mboutin2/working-directory/project/node_modules/babel-register/lib/node.js:166:7)
at Module.load (module.js:357:32)
at Function.Module._load (module.js:314:12)```

**Comment by RakanNimer** [comment]
Hey, yes this a known limitation for now : https://github.com/RakanNimer/react-google-charts#isomorphic-support. PRs are welcome!

**Comment by DWboutin** [comment]
Great, i'll check what i can do!

**Comment by voogryk** [comment]
I fix it and create PR https://github.com/RakanNimer/react-google-charts/pull/65 Enjoy!)

**Comment by RakanNimer** [comment]
Merged #65 should be fixed.

**Comment by nicholasalanbrown** [comment]
I want to use this package to create chart images on my server with Phantom.js. I have everything working server side except when I try and run ReactDOMServer.renderToString()' I just get '"creating chart...". How would I wait until the chart is ready on the server before working with its HTML?
